forc/src/ops/forc_init.rs

```diff
@@ -82,6 +82,10 @@ fn print_welcome_message() {
 pub fn init(command: InitCommand) -> Result<()> {
     let project_name = command.project_name;
     validate_name(&project_name, "project name")?;
+    let project_type = match command.project_type {
```

Comment:
[structopt has an option for a default value](https://docs.rs/structopt/latest/structopt/#how-to-derivestructopt).

You can [also make the lowercasing part of structopt as well](https://docs.rs/structopt/latest/structopt/#custom-string-parsers)